model/browser/events.yaml

```diff
@@ -0,0 +1,104 @@
+groups:
+  - id: event.browser.resource_timing
+    stability: development
+    type: event
+    name: browser.resource_timing
+    brief: >
+      This event describes the timing metrics as provided by PerformanceResourceTiming Performance API.
+      These metrics are related to fetching a resource, such as
+      XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+    note: >
+      This event captures data from the [ResourceTiming](https://www.w3.org/TR/resource-timing/).
+      It is recommended to be captured by using the PerformanceResourceTiming API.
+      If the page starts unloading before the page loads,
+      then the partial data can be captured by calling the [performance.getEntriesByType](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType) method.
+    attributes:
+      - ref: url.full
+        requirement_level: required
+        note: >
+          Generally the url doesn't include any query strings or any leading credentials,
```

Comment:
@open-telemetry/specs-semconv-maintainers do you see any issue with using `url.full` for something that doesn't include the query string?